people/people.go

```diff
@@ -0,0 +1,43 @@
+package people
+
+import (
+	"fmt"
+	"sort"
+)
+
+type People []Person
+
+func (pp People) Append(ps Person) People {
```

Comment:
What is the reason of adding this wrapper ?

Reply:
The rudimentary method from the old version of the code (checked errors from Person, no longer relevant). Left for backward compatibility, but can be removed (as done in tests)